libs/bixarena/api-description/openapi/api-public.openapi.yaml

```diff
@@ -48,6 +48,7 @@ paths:
           $ref: '#/components/responses/Forbidden'
   /battles:
     get:
+      x-anonymous-access: true
```

Comment:
This vendor extension has been applied to almost all the endpoints of the API service so that the API gateway will allow the rerouting to occur until we have finalized the integration of the auth service with the API gateway. Once done, these anonymous vendor extensions will be removed from the endpoints that require a JWT, that is, the endpoints that require the user to be authenticated.